client/components/theme-type-badge/index.tsx

```diff
@@ -19,56 +19,51 @@ import './style.scss';
 
 interface Props {
 	canGoToCheckout?: boolean;
-	forcePremium?: boolean;
+	isLockedStyleVariation?: boolean;
```

Comment:
Maybe we could change the name of this variable so that it's more similar to our product wording. I think a good name could be `isUpgradeRequiredForStyleVariation `

Reply:
I considered that too, but couldn't think of a better name than `isLockedStyleVariation`.

To me `isUpgradeRequiredForStyleVariation` infers that the site needs an upgrade to use some style variations, but it's not clear whether the badge is currently used for those variations or not and if you'd need to check something else. A more clear name would be `isStyleVariationThatRequiresUpgrade` but I prefer `isLockedStyleVariation` because it's shorter.

By using `isLockedStyleVariation` as name, I'm explicitly telling the component that the badge will be for a style variation that needs to be unlocked by upgrading. And we're using the lock/unlock terminology in the UX, so it's aligned to some extent.